seacrowd/sea_datasets/my_paraphrase/my_paraphrase.py

```diff
@@ -0,0 +1,199 @@
+        "https://github.com/ye-kyaw-thu/myParaphrase/raw/main/corpus/ver1.0/csv-qqp/open-test.final.manual.csv",
+        "https://github.com/ye-kyaw-thu/myParaphrase/raw/main/corpus/ver1.0/csv-qqp/closed-test.csv",
+    ],
+}
+
+_SUPPORTED_TASKS = [Tasks.PARAPHRASING]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+_TAGS = [0, 1]
+
+
+class MyParaphraseDataset(datasets.GeneratorBasedBuilder):
+    """The "myParaphrase" corpus is a Burmese dataset used for paraphrase identification. \
+    It includes 40,461 training pairs and 2,000 test pairs. Pairs are labeled "1" for paraphrases and "0" otherwise."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+    SEACROWD_SCHEMA_NAME = "t2t"
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_source",  # source
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="paraphrase_source",
+            subset_id=f"{_DATASETNAME}_paraphrase",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_seacrowd_{SEACROWD_SCHEMA_NAME}",  # schema
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema=f"seacrowd_paraphrase_{SEACROWD_SCHEMA_NAME}",
+            subset_id=f"{_DATASETNAME}_paraphrase",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_non_paraphrase_source",  # source
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema="non_paraphrase_source",
+            subset_id=f"{_DATASETNAME}_non_paraphrase",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_non_paraphrase_seacrowd_{SEACROWD_SCHEMA_NAME}",  # schema
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema=f"seacrowd_non_paraphrase_{SEACROWD_SCHEMA_NAME}",
+            subset_id=f"{_DATASETNAME}_non_paraphrase",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_all_source",  # source
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="all_source",
+            subset_id=f"{_DATASETNAME}_all",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_all_seacrowd_{SEACROWD_SCHEMA_NAME}",  # schema
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema=f"seacrowd_all_{SEACROWD_SCHEMA_NAME}",
+            subset_id=f"{_DATASETNAME}_all",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_seacrowd_{SEACROWD_SCHEMA_NAME}_paraphrase"
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema in [
+            "paraphrase_source",
+            "non_paraphrase_source",
+            "all_source",
+            # "source"
+        ]:
```

Comment:
suggestion (optional): consider changing the detection to following
```suggestion
        if self.config.schema.endswith("_source"):
```